docs/02-app/01-building-your-application/04-caching/index.mdx

```diff
@@ -0,0 +1,589 @@
+
+### Opting out
+
+To opt out of memoization in `fetch` requests, you can pass an `AbortController` `signal` to the request.
+
+```js
+const { signal } = new AbortController()
+fetch(url, { signal })
+```
+
+Since you manually opt into using the React `cache` function, there's no way to opt out.
+
+## Data Cache
+
+Next.js has a built-in Data Cache that **persists** the result of data fetches across incoming **server requests** and **deployments**. This is possible because Next.js extends the native `fetch` API to allow each request on the server to set its own persistent caching semantics.
+
+By default, data requests that use `fetch` are **cached**. You can use the [`cache`](#fetch-optionscache) and [`next.revalidate`](#fetch-optionsnextrevalidate) options of `fetch` to configure the caching behavior.
+
+> **Good to know**: In the browser, the `cache` option of `fetch` indicates how a request will interact with the browser's HTTP cache, with Next.js, the `cache` option indicates how a server-side request will interact with the servers Data Cache.
+
+<Image
+  alt="Diagram showing how cached and uncached fetch requests interact with the Data Cache. Cached requests are stored in the Data Cache, and memoized, uncached requests are fetched from the data source, not stored in the Data Cache, and memoized."
+  srcLight="/docs/light/data-cache.png"
+  srcDark="/docs/dark/data-cache.png"
+  width="1600"
+  height="679"
+/>
+
+The first time a `fetch` request is called during rendering, Next.js checks the Data Cache for a cached response. If a cached response is found, it's returned immediately and [memoized](#request-memoization). If not, the request is made to the data source, the result is stored in the Data Cache, and memoized.
+
+For uncached data (e.g. `{ cache: no store }`), the result is always fetched from the data source, and memoized.
+
+> **Differences between the Data Cache and Request Memoization**
+>
+> While both caching mechanisms help improve performance by re-using cached data, the Data Cache is persistent across incoming requests and deployments, whereas memoization only lasts the lifetime of a request.
+>
+> With memoization, we reduce the number of **duplicate** requests in the same render pass that have to cross the network boundary from the rendering server to the Data Cache server (e.g. a CDN or Edge Network) or data source (e.g. a database or CMS). With the Data Cache, we reduce the number of requests made to our origin data source.
+
+### Duration
+
+The Data Cache is persistent across incoming requests and deployments unless you revalidate or opt-out.
+
+### Revalidating
+
+Cached data can be revalidated in two ways, with:
+
+- **Time-based Revalidation**: Automatically revalidate data after a certain amount of time has passed. This is useful for data that changes infrequently and freshness is not as critical.
```

Comment:
thought: I think the phrase `Automatically revalidate data after a certain amount of time has passed` is misleading. It again implies (to me) that no matter what, after the time has passed, data will be up-to-date in the cache, even if no request happened in that time frame, and that the next request after that timeframe will already receive fresh data.

It becomes clearer after reading the **How Time-based Revalidation Works**, especially:

> 3.  After the timeframe, the next request will still return the cached (now stale) data.
    - Next.js will trigger a revalidation of the data in the background.

But I think it would be better to clarify this up-front.

Reply:
Good point, I'll update it! 🙏🏼